spec/factories/shipping_rate_factory.rb

```diff
@@ -0,0 +1,14 @@
+# Included for backward compatability with Solidus version < v1.2
```

Comment:
There's the `solidus_support` Gem for finding out Solidus version numbers in 1.1 and 1.2 now